lib/bundle-validator.js

```diff
@@ -106,6 +107,119 @@ function validateThemeConfiguration(callback) {
     callback(null, true);
 }
 
+/**
+ * Find translatable strings
+ * @param {object} schema
+ * @param {function} callback
+ */
+function getTranslatableStrings(schema, callback) {
+    var trackedKeys = ['name', 'content', 'label', 'settings', 'options'];
```

Comment:
I think we may also want to track the `group` option.

https://github.com/bigcommerce/cornerstone/blob/master/schema.json#L54

Reply:
I think we should use const instead of var if possible.

Reply:
Could you please provide more information on the use of the `group` option? I didn't find any usage.

Reply:
I can find examples in cornerstone schema, but it seems like there is a bug where store design is not displaying the grouped selects. Let me bring this back to the team and see what the desired state should be.

Reply:
I added `group` options into `trackedKeys`